sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -57,8 +57,8 @@ class SparseEncoder(SentenceTransformer):
         local_files_only (bool, optional): Whether or not to only look at local files (i.e., do not try to download the model).
         token (bool or str, optional): Hugging Face authentication token to download private models.
         use_auth_token (bool or str, optional): Deprecated argument. Please use `token` instead.
-        truncate_dim (int, optional): The dimension to truncate sentence embeddings to. `None` does no truncation. Truncation is
-            only applicable during inference when :meth:`SparseEncoder.encode` is called.
+        max_active_dims (int, optional): The maximum number of active dimensions in the output of the model. Defaults to None. This mean there will be no
+            limit on the number of active dimensions and can be risky when indexing the embeddings.
```

Comment:
```suggestion
        max_active_dims (int, optional): The maximum number of active (non-zero) dimensions in the output of the model. Defaults to None. This mean there will be no
            limit on the number of active dimensions and can be risky when indexing the embeddings.
```
Maybe instead of "risky" we can be a bit more explicit? E.g. "can be slow or memory-intensive if your model wasn't (yet) finetuned to high sparsity" or something? I assume that's what you're trying to say here